google-cloud-core/lib/google/cloud/config.rb

```diff
@@ -495,16 +493,8 @@ def nil?
       # @private a list of key names that are technically illegal because
       # they clash with method names.
       #
-      ILLEGAL_KEYS = %i[
-        add_options
-        initialize
-        instance_eval
-        instance_exec
-        method_missing
-        singleton_method_added
-        singleton_method_removed
-        singleton_method_undefined
-      ].freeze
+      ILLEGAL_KEYS = [:add_options, :initialize, :instance_eval, :instance_exec, :method_missing,
```

Comment:
I'm also not a huge fan of the `%` list sigils in general, but I do feel the original formatting was more readable.

Reply:
Me too. This was discussed and approved in googleapis/ruby-style#14.

Reply:
Oops, that was for a different change. This might be a change in Rubocop's default rules.

Reply:
I guess I meant the layout. Does the style allow us to do something like:

```ruby
ILLEGAL_KEYS = [
  :add_options,
  :initialize,
  # ...
].freeze
```

Reply:
Yes, it does. This format is the one that Rubocop uses when converting from the `%` list. I have updated this array to be indented like you listed.